submissions/BlueLamantine/js-dom/dom.js

```diff
@@ -0,0 +1,148 @@
+	id: 'americano',
+	classes: {
+		bgColor: 'americano-bg',
+		textColor: 'cappuccino-text'
+	},
+	paragraph: 'Shots of espresso diluted with water.',
+}, {
+	title: 'Macchiato',
+	id: 'macchiato',
+	classes: {
+		bgColor: 'macchiato-bg',
+		textColor: 'cappuccino-text'
+	},
+	paragraph: 'Espresso with a dash of frothy foamed milk.',
+}, {
+	title: 'Flat white',
+	id: 'flatw',
+	classes: {
+		bgColor: 'flatw-bg',
+		textColor: 'americano-text'
+	},
+	paragraph: 'Double shot of espresso with steamed milk.',
+}, {
+	title: 'Cappuccino',
+	id: 'cappuccino',
+	classes: {
+		bgColor: 'cappuccino-bg',
+		textColor: 'americano-text'
+	},
+	paragraph: 'Double shot of espresso topped with an airy thick layer of foamed milk.',
+}, {
+	title: 'Caffe latte',
+	id: 'latte',
+	classes: {
+		bgColor: 'latte-bg',
+		textColor: 'doppio-text'
+	},
+	paragraph: 'A shot of espresso in steamed milk lightly topped with foam.',
+}];
+const MENU = document.getElementById('mainMenu');
+const CUP = document.getElementById('coffeeCups');
+const ART = document.getElementById('articles');
+
+function render() {
+	COFFEE.forEach(element => {
+		MENU.appendChild(createListNode(element));
+		CUP.appendChild(createCupNode(element));
+		ART.appendChild(createContentNode(element));
+	});
+};
+
+function createListNode(el) {
+	let listItem = document.createElement('li');
+	listItem.className = `menu-item ${el.classes.bgColor}`;
+	listItem.innerHTML = `
+    <a href="#" id="${el.id}" class="item__link ${el.classes.textColor}">${el.title}</a>
+    `;
+	if (el.id == 'doppio') {
+		listItem.classList.add('active');
+	}
+	return listItem;
+};
+
+function createCupNode(el) {
+	let cup = document.createElement('div');
+	cup.id = `${el.id}-cup`;
+	cup.className = `cup ${el.id}-cup__bg `;
+	if (el.id !== 'doppio') {
+		cup.hidden = true;
+	}
+	cup.innerHTML = `
+        <div class = "paw-left"></div>
+        <div class = "paw-right"></div>
+    `;
+	return cup;
+};
+
+function createContentNode(el) {
+	const articleContent = document.createElement('article');
+	const headerContent = document.createElement('h2');
+	const textContent = document.createElement('p');
+	articleContent.id = `${el.id}-article`;
+	articleContent.className = 'description';
+	if (el.id !== 'doppio') {
+		articleContent.hidden = true;
+	}
+	headerContent.innerHTML = el.title;
+	textContent.innerHTML = el.paragraph;
+	articleContent.appendChild(headerContent);
+	articleContent.appendChild(textContent);
+	return articleContent;
+};
+
+function toggleMenuItem() {
+	MENU.addEventListener('click', e => {
+		e.preventDefault();
+		const selectedMenuItem = e.target;
+		if (selectedMenuItem.matches('.item__link')) {
+			const currentMenuItem = document.querySelector('.menu-item.active');
+			currentMenuItem.classList.remove('active');
+			selectedMenuItem.closest('li').classList.add('active');
+			toggleContent(selectedMenuItem.id);
+		}
+	});
+};
+
+function toggleContent(idEL, arr) {
+	const content = {
```

Comment:
It seems we didn't get ourselves in the previous comments
Look, now you're trying to hide two sections which depend on one cup object.
Why do we separate information about ONE type of coffee into TWO separate blocks? Let's make it simpler. You can query only the wrap of article and cup and make it visible and hidden.
It helps you to reduce the amount of similar code in this function.
Let me know if you got me.

Reply:
OK. Now it is clear! :grin:
Now I have wrapper of cups and articles and hide it with class ```.hidden{dislpay:none}```, not separately like it was earlier. 
And DOM creates this wraps-blocks (blocks are does not exist in initial html file).